db/src/main/kotlin/edu/artic/db/daos/ArticExhibitionCMSDao.kt

```diff
@@ -0,0 +1,21 @@
+package edu.artic.db.daos
+
+import android.arch.persistence.room.Dao
+import android.arch.persistence.room.Insert
+import android.arch.persistence.room.OnConflictStrategy
+import android.arch.persistence.room.Query
+import edu.artic.db.models.ArticExhibitionCMS
+import io.reactivex.Flowable
+
+@Dao
+interface ArticExhibitionCMSDao {
+
+    @Query("DELETE FROM ArticExhibitionCMS")
+    fun clear()
+
+    @Query("select * from ArticExhibitionCMS")
```

Comment:
one is caps other is lowercase. choose a SQLite syntax pattern and stick with it.

Reply:
Updated all of them to lower case.